hack/update-codegen.sh

```diff
@@ -19,7 +19,14 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 $GOPATH/src/github.com/kubernetes/code-generator 2>/dev/null || echo ../code-generator)}
+if [ ! -d "$GOPATH/src/github.com/kubernetes/code-generator" ]
+then
+    wget https://github.com/kubernetes/code-generator/archive/v0.16.4.tar.gz -P /tmp
```

Comment:
why not to include it in vendor ?
have a look at tools.go, it includes packages that are required for build/test only:

```
// +build tools

package tools

import (
        _ "github.com/aktau/github-release"
        _ "github.com/onsi/ginkgo/ginkgo"
        _ "github.com/operator-framework/operator-sdk/cmd/operator-sdk"
        _ "golang.org/x/tools/cmd/goimports"
)
```

Reply:
I was unable to download that tool with any of go tools. It's always:
`build k8s.io/code-generator: cannot load k8s.io/code-generator: no Go source files`